src/appfl/logger/client_logger.py

```diff
@@ -45,16 +45,23 @@ def __init__(
             if not os.path.exists(file_dir):
                 pathlib.Path(file_dir).mkdir(parents=True, exist_ok=True)
             real_file_name = f"{file_dir}/{file_name}.txt"
+            # check if the file exists
+            file_exists = os.path.exists(real_file_name)
             f_handler = logging.FileHandler(real_file_name)
             f_handler.setLevel(logging.INFO)
             f_handler.setFormatter(fmt)
             self.logger.addHandler(f_handler)
-            self.logger.info(f"Logging to {real_file_name}")
+            if not file_exists:
```

Comment:
Does this mean if the `real_file_name` does not exist, we are logging that information with the logger?

Reply:
`f_handler = logging.FileHandler(real_file_name)` will create the `real_file_name` if it does not exist, and `file_exists` checks before that to see if that file exists.

If not, which means we are logging to such file for the first time, then log "Logging to..."
If the file exists, which means we are continuing logging to an existing log file, so we choose not to log "Logging to..." again